backend/application/syson-application/src/test/java/org/eclipse/syson/application/export/ImportExportTests.java

```diff
@@ -70,6 +70,38 @@ public void checkSuccessionAsUsageImplicitSourceTest() throws IOException {
         this.checker.check(input, input);
     }
 
+    @DisplayName("Given a SuccessionAsUsage with target defined after the then keyword, when importing and exporting the model,  then the exported text file should be semantically equal.")
+    @Test
```

Comment:
Please add `@Test` before `@DisplayName` (valid for every other test in this PR and in the future)

Reply:
Done